p2p-media-loader-demo/tsconfig.json

```diff
@@ -11,8 +11,8 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "strict": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
```

Comment:
It's really annoying when your IDE highlights in red color every single unused variable while you are just writing test code. Especially since we have corresponding eslint rule that treats unused variable as warning. But if you find it safer, okey.